### PR TITLE
Create consistent ui/input and ui/display for Cell and Fields type : …

### LINKED ISSUE #1562: Create consistent ui/input and ui/display for Cell and Fields type : Probability, DoubleText, DoubleTextChip
We already refactored into ui/inputs and ui/displays for multiple types : Date, Boolean, Phone, Email, Money, Number.

We still need to apply the same process to Probability, DoubleText and DoubleTextChip.

You'll find examples here :
- `ui/input/components/DateInput`
- `ui/input/components/TextInput`
- `ui/content-display/components/DateDisplay`
- `ui/content-display/components/TextDisplay`

It should be applied in Cell and Field components.

We will work on the Relation type in a separate issue.

### PR DESCRIPTION
**Pull request by gitstart-twenty**
Fixes #1562

### CODE REVIEW

**Comment by lucasbordeau** [comment]
This PR should implement : 

- `ui/input/components/ProbabilityInput` (no content-display counter part, see BooleanInput.tsx)
- `ui/input/components/DoubleTextInput`
- `ui/input/components/DoubleTextChipInput`
- `ui/content-display/components/DoubleTextDisplay`
- `ui/content-display/components/DoubleTextChipDisplay`

Those components should be the unique way to input and display those types in table cell and editable fields. (Except for People page title, which is a specific component)

**Comment by gitstart-twenty** [comment]
> This PR should implement :
> 
> * `ui/input/components/ProbabilityInput` (no content-display counter part, see BooleanInput.tsx)
> * `ui/input/components/DoubleTextInput`
> * `ui/input/components/DoubleTextChipInput`
> * `ui/content-display/components/DoubleTextDisplay`
> * `ui/content-display/components/DoubleTextChipDisplay`
> 
> Those components should be the unique way to input and display those types in table cell and editable fields. (Except for People page title, which is a specific component)

Thanks for the clarification @lucasbordeau

**Comment by gitstart-twenty** [comment]
Hey @lucasbordeau,
Are the new changes more aligned towards the end goal for the task?

**Comment by charlesBochet** [comment]
@lucasbordeau whenever you can take a look :)

**Review by lucasbordeau — CHANGES_REQUESTED** [review]

File: front/src/modules/ui/content-display/components/DoubleTextChipDisplay.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        <PersonChip
+          id={currentRowEntityId ?? ''}
+          name={displayName}
+          pictureUrl={avatarUrlValue}
+        />
+      );
+    }
+    default:
+      console.warn(
+        `Unknown relation type: "${entityType}" in GenericEditableDoubleTextChipCellDisplayMode`,
```
[the remark this answers is not in the document]
Comment (gitstart-twenty) [reply front/src/modules/ui/content-display/components/DoubleTextChipDisplay.tsx]: Hey @lucasbordeau,
The above comments have been addressed.

**Review by lucasbordeau — APPROVED** [review]

**Pull request merged**